### PR TITLE
Adjusting setup instructions

### PR DESCRIPTION
**Pull request by acorncom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,11 @@ This Ember Application allows you to browse API docs.
   npm install && bower install
   ```
 
-2. Install `yuidocjs` (if needed) on your machine (`npm -g install yuidocjs`) and run it in the root of your ember.js folder (`yuidoc .`)
+2. Clone the main `ember.js` repo down to your machine, cd into it and run `bower install` to install all needed dependencies.
 
-1. Start the app with the following command args
+3. Install `yuidocjs` (if needed) on your machine (`npm -g install yuidocjs`) and run it in the root of your ember.js folder (`yuidoc .`)
```
Comment (rwjblue) [inline README.md]: I'm not sure if this is actually needed. In ember-cli-yuidoc-converter we depend directly on YUIDoc and use that local version for compiling docs. So I'm not sure we need this globally installed. 

Can you test?

Comment (camthompson) [inline README.md]: The instructions in this PR worked for me. I needed YUIDoc installed globally in order to run it in the ember.js directory.

Comment (acorncom) [inline README.md]: @rwjblue Confirmed, we don't need yuidocjs installed globally, I'll take that out.

**Comment by acorncom** [comment]
@rwjblue The instructions here are updated.

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**